Snippets/Core/Core_6/Scanning/ScanningPublicApi.cs

```diff
@@ -44,5 +52,28 @@ void ScanningUpgrade(EndpointConfiguration endpointConfiguration)
             #endregion
         }
 
+        void ScanningApDomainAssemblies(EndpointConfiguration endpointConfiguration)
+        {
+            #region ScanningApDomainAssemblies
+
+            endpointConfiguration.AssemblyScanner(scannerConfiguration =>
+            {
+                scannerConfiguration.ScanAppDomainAssemblies = true;
+            });
+
+            #endregion
+        }
+
+        void SwallowScanningExceptions(EndpointConfiguration endpointConfiguration)
+        {
+            #region SwallowScanningExceptions
+
+            endpointConfiguration.AssemblyScanner(scannerConfiguration =>
+            {
+                scannerConfiguration.ThrowExceptions = false;
```

Comment:
yes, "back then", the "lambda approach" was considered to be the way to do it. I'm considering to switch the API to the guidelines as it's new in 6.2